shell/src/main/java/org/apache/accumulo/shell/commands/TableOperation.java

```diff
@@ -73,24 +73,12 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
       Shell.log.warn("No tables found that match your criteria");
     }
 
-    boolean more = true;
-    // flush the tables
+    // do op if forced or user answers prompt with yes
     for (String tableName : tableSet) {
-      if (!more) {
-        break;
-      }
```

Comment:
The removal of this logic is a slight change in behavior. That's not necessarily a bad thing, but in this case, I think it's worth preserving the current behavior. The previous code would interpret a EOF (triggered by Ctrl-C, or perhaps Ctrl-D ? to abort the prompt) as a "stop asking for the rest" instruction. With the behavior in this PR, the confirm method will return false, and continue asking about the rest of the tables.

To solve this, instead of having the confirm method return a boolean, it could return an `Optional<Boolean>`, with not present generally meaning "no response" or "abort the action that triggered the question", which could be used to support this case, or other similar cases.